runtime/vm/jvminit.c

```diff
@@ -6349,6 +6350,15 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 	Xj9BreakPoint("jvminit");
 #endif
 
+        {
```

Comment:
This should indent via tabs to match the surrounding code.

Reply:
bah, i thought I had caught all those. Thanks!

Reply:
The opening brace should be indented one tab less than the block it begins (unlike JIT code).